Frends.AzureBlobStorage.DeleteContainer/README.md

```diff
@@ -0,0 +1,28 @@
+# Frends.AzureBlobStorage.DeleteContainer
+FRENDS Task for Deleting a Container from a blob in Azure.
+
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](https://opensource.org/licenses/MIT) 
+[![Build](https://github.com/FrendsPlatform/Frends.AzureBlobStorage/actions/workflows/DeleteContainer_build_and_test_on_main.yml/badge.svg)](https://github.com/FrendsPlatform/Frends.AzureBlobStorage/actions)
+![MyGet](https://img.shields.io/myget/frends-tasks/v/Frends.AzureBlobStorage.DeleteContainer)
+![Coverage](https://app-github-custom-badges.azurewebsites.net/Badge?key=FrendsPlatform/Frends.AzureBlobStorage/Frends.AzureBlobStorage.DeleteContainer|main)
+
+Returns boolean of action's success.
```

Comment:
You don't need this line since documentation about results will be available in Task Portal.